cmd/cp-url-syntax.go

```diff
@@ -42,7 +43,7 @@ func checkCopySyntax(ctx *cli.Context) {
 	for _, srcURL := range srcURLs {
 		_, _, err := url2Stat(srcURL)
 		if err != nil {
-			fatalIf(err.Trace(srcURL), fmt.Sprintf("Unable to stat '%s'.", srcURL))
+			console.Fatalf("Source %s doesn't exist.\n", srcURL)
```

Comment:
Couldn't this be sometimes a network problem ?

We probably can find a better phrase than 'Unable to stat' of this latter is considered not user friendly